app/assets/stylesheets/scaffolds.scss

```diff
@@ -37,15 +37,15 @@ body {
 
 .zen-mode {
   .top-bar,
-  .primary-sticky-nav,
-  .article-actions {
+  .crayons-article-sticky,
+  .crayons-article-actions {
     display: none !important;
```

Comment:
Do we still need the `important!`?

Reply:
Good question, I don't think we do. I'll check and remove it if it's not needed anymore.